docs/content/management-decision-logs.md

```diff
@@ -253,6 +253,38 @@ to track **remove** vs **upsert** mask operations.
 }
 ```
 
+### Drop Decision Logs
+
+Drop rules filters all decisions, which evaluate to `true`, before logging them.
+
+This rule will drop all requests to the _allow_ rule in the _kafka_ package, that returned _true_:
+```live:drop_rule_example/kafka_allow_rule:module:read_only
+package system.log
+
+drop {
+  input.path == "kafka/allow"
+  input.result
```

Comment:
I'd prefer an explicit `input.result == true` here.

Reply:
Is that the intention, though? I've read that as "is defined"

Reply:
My bad. You're right, let's update this.